cmd/export_accounts.go

```diff
@@ -60,15 +63,25 @@ the export_ledger_entry_changes command.`,
 			}
 
 			if !useStdout {
-				outFile.Write(marshalled)
+				nb, err := outFile.Write(marshalled)
+				numBytes += nb
 				outFile.WriteString("\n")
+				if err != nil {
+					cmdLogger.Info("Error writing accounts to file: ", err)
+				}
```

Comment:
💡  moving the `if err != nil` line up. Generally that check comes right after the line the `err` is returned. I don't think it matters much here for functionality but helps for readability
```suggestion
				nb, err := outFile.Write(marshalled)
				if err != nil {
					cmdLogger.Info("Error writing accounts to file: ", err)
				}
				numBytes += nb
				outFile.WriteString("\n")
				
```

Reply:
I think there are a couple spots below that are similar

Reply:
thanks, i'll make the necessary updates to each `cmd_*` script